### PR TITLE
Pop-up details window

### PR DESCRIPTION
**Pull request by lengushuru**
**changes made description**

-  Kindly make sure your buttons' images are displaying properly. I am unable to view them on the popup- popup images added to the images folder.
- pop-up changed to render data correctly.
- corrected conflict with main branch

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# Status: INVALID PULL REQUEST
## Hi @lengushuru :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
Invalid Code Review Request

### Your project is Invalid :no_entry: 
- [ ] You have to use updated Microverse readme
    - Kindly use the updated version of the microverse [readme](https://github.com/microverseinc/readme-template) template for your readme, this is a requirement to get a review
- [ ]  Linter result should be green, this means that the last commit have a green check mark (✅) or a summary of the passed checks is in the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

### Cheers and Happy coding! :clap: :clap: :clap:

---

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @TuyishimireEric_

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Review by Ismailco — COMMENTED** [review]
Hi @lengushuru, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an old README, kindly consider using [the new README template](https://github.com/microverseinc/readme-template) provided by microverse


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: scripts.js [hunk 1]
```diff
@@ -15,4 +15,251 @@ const navLinks = document.querySelectorAll('.nav-item');
...
+  cardButton1.appendChild(btntext);
+  btnDiv.appendChild(liveLink);
+
+  const source = document.createElement('a');
+  source.href = card.cardButton2;
+  const cardButton2 = document.createElement('div');
+  cardButton2.className = 'see-more';
+  source.appendChild(cardButton2);
+  const btnimg2 = document.createElement('img');
+  btnimg2.src = './images/btn.svg';
```
Comment (bienvenuushindi) [inline scripts.js:242]: :red_circle: [Requested Change]  Kindly make sure your buttons' images are displaying properly. I am unable to view them on the popup
This is what I mean :arrow_down: 

Your Design :paintbrush: 
popuimage

Figma Design :paintbrush: 
btnfigma

File: scripts.js [hunk 2]
```diff
@@ -15,4 +15,251 @@ const navLinks = document.querySelectorAll('.nav-item');
...
+
+    if (j < card.list1.length - 2) {
+      const list1items = document.createElement('l1');
+      list1items.className = 'dot';
+      const dotz = document.createElement('img');
+      dotz.src = li4;
+      list1items.appendChild(dotz);
+      list1.appendChild(list1items);
+    }
+  }
```
Comment (bienvenuushindi) [inline scripts.js:204]: :red_circle: [Required] 
Please Make sure when rendering the pop up you ship the right content for each item. As your pop-up keeps the same data every time it pops in. You should update the state of each part depending on the source item's button. The goal of this is to associate each project with the popup window details
This is actually what is happening :arrow_down: 

Peek 2022-11-26 14-09

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @lengushuru ,

Good job implementing most of the required changes!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔

## (Changes Required) Aspects to improve: ♻️
- [x] Kindly fix the conflicts on this branch 

- [x] Please check the inline comments below.


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mckent05 — APPROVED** [review]
Hi @lengushuru,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved



## Highlights
- All linter checks are passing ✔️ 
- Popup window works as expected 👏 
- Your PR is professional 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**